pkgs/development/tools/electron/update.py

```diff
@@ -0,0 +1,227 @@
+#! /usr/bin/env nix-shell
+#! nix-shell -i python -p python3 nix nix-universal-prefetch prefetch-yarn-deps prefetch-npm-deps
+
+import traceback
+import csv
+import base64
+import os
+import re
+import tempfile
+import subprocess
+import json
+import sys
+from codecs import iterdecode
+from datetime import datetime
+from urllib.request import urlopen
+
+depot_tools_checkout = tempfile.TemporaryDirectory()
+subprocess.check_call([
+    "nix-prefetch-git",
+    "--builder", "--quiet",
+    "--url", "https://chromium.googlesource.com/chromium/tools/depot_tools",
+    "--out", depot_tools_checkout.name,
+    "--rev", "543fdf32d15fe47a0beadc3f5a3e0d6218ad237c"])
```

Comment:
Is that a hardcoded rev?

Reply:
Yes, it is a known working rev. But should be independent of the chromium version.